apps/site/pages/docs/faststore-config-options/custom-cypress-test.mdx

```diff
@@ -64,3 +64,59 @@ Like any other project using Cypress, you'll follow standard practices, includin
 ![checks-integration-tests](https://vtexhelp.vtexassets.com/assets/docs/src/custom-test-checks___583ac9b78e43ebdafde93a735a47d61b.png)
 
 6. Review the results to check the execution status of your custom integration tests.
+
+---
+
+## Exclusive execution of your custom Cypress tests
+
+With continuous development, some storefront projects may have changes in the components and/or pages's code that cause errors in the out of the box end-to-end tests that are run for all FastStore projects.
```

Comment:
Due to ongoing development, some storefront projects may have code changes in their components and/or pages. These changes can lead to errors in the out-of-the-box end-to-end tests that run for all FastStore projects.